src/lib/marquee.tsx

```diff
@@ -64,8 +64,8 @@ export function Marquee({
     // to give us the ammount of items needed to fill the container.
     let neededAmount = Math.ceil(containerWidth / marqueeWidth) * 2 - 1
 
-    // check if needed ammount if less that one, if it is set to be just 1.
-    if (neededAmount < 1 || isNaN(neededAmount)) {
+    // check if needed amount is less than one, infininity or not a valid number, setting the value to be just 1.
+    if (neededAmount < 1 || isNaN(neededAmount) || neededAmount === Infinity) {
```

Comment:
```suggestion
    if (neededAmount < 1 || isNaN(neededAmount) || neededAmount === Infinity) {
```
Nit: This comment isn't necessary. Your `if` statement explains it perfectly.

Reply:
For sho, removed!